tools/glib.supp

```diff
@@ -1,3 +1,12 @@
+# This GLib suppressions file is known to be used at least by
```

Comment:
How about something like
```
This GLib suppressions file is known to be also used by:
* rpm-software-management/libhif (https://github.com/rpm-software-management/libhif)
```

What do you mean by "look for any changes there"? That if something is missing we should look there, too?